test/src/main/configurations/MainConfig/ConfigurationMessageStoreSenderAndListenerPeek.xml

```diff
@@ -0,0 +1,39 @@
+<module>
+	<adapter name="MessageStoreListener">
+		<receiver className="nl.nn.adapterframework.receivers.GenericReceiver" name="MessageStoreListener" transactionAttribute="Required" pollInterval="1">
+			<listener className="nl.nn.adapterframework.jdbc.MessageStoreListener" jmsRealm="jdbc" name="MessageStoreListener" slotId="${applicationId}/MessageStoreListener" peekUntransacted="true" />
+		</receiver>
```

Comment:
Er is ook al een ConfigurationMessageStoreSenderAndListener. Is deze test/configuratie te combineren daarmee?